opacus/privacy_engine.py

```diff
@@ -53,8 +53,11 @@ def __init__(self, accountant: str = "rdp", secure_mode=False):
 
         """
         self.accountant = get_accountant(mechanism=accountant)
```

Comment:
I do like the arg in constructor being called just `accountant`, but we use `accountant_mechanism` elsewhere. Do you think it is useful to make this consistent?

Reply:
Is there a missing "not"? If yes, then I agree :)

Reply:
No, there is no missing "not" :)

Reply:
Oh, I see what you mean
You like both consistency and args being called "accountant"

The inconsistency I wanted to address here is that some `accountant` parameters/variables are strings, others are actual accountant objects. Are we ok with this?

Reply:
Yeah. Consistency would mean changing the arg name in PrivacyEngine constructor to `accountant_mechanism`. that way, all accountant objects will be `accountant` and strings will be `accountant_mechanism`

Reply:
It's a toughie - I also like args being called `accountant`.
Does consistency here overweights clear arg name for you? After all, consistency is for the internal implementation, but constructor arg is for everyone

Reply:
Are you sure you don't like it the way it's implemented now? It's always `accountant: str` when it's a parameter and it's always `self.accountant: IAccountant` when it's an attributed.
all `*_mechanism` is moved to accountant object as you suggested

Reply:
Sorry, I missed replying yesterday. 

> It's a toughie - I also like args being called accountant.

+1

> Does consistency here overweights clear arg name for you? 

Doesn't overweigh, probably equal. Hence the equivocality. :/

> Are you sure you don't like it the way it's implemented now?

I think I can live with it. Should it ever get confusing, I would propose changing `self.accountant` to `self.accountant_obj` or some such thing. I like `accountant: str` as an arg. If no one else feels strongly, I vote for keeping things as is for now.